retrofit/src/test/java/retrofit/RestAdapterTest.java

```diff
@@ -129,4 +143,62 @@
           + "Checked: [Built-in CallAdapterFactory]");
     }
   }
+
+  @Test public void defaultConverterThrowsSerializing() throws IOException {
+    RestAdapter ra = new RestAdapter.Builder()
+        .endpoint(server.getUrl("/").toString())
+        .build();
+    CallMethod example = ra.create(CallMethod.class);
+    Call<String> call = example.disallowed("Hi!");
+    try {
+      call.execute();
+      fail();
+    } catch (UnsupportedOperationException e) {
+      assertThat(e).hasMessage("No converter installed to handle serializing class java.lang.String to body");
```

Comment:
I think the words here are flipped? 

_No converter installed to handle serializing class java.lang.String to body_
vs.
_No converter installed to handle serializing body to class java.lang.String_